measurements/api/measurements.py

```diff
@@ -361,26 +423,83 @@ def list_measurements(
     if order_by is not None:
         q = q.order_by(text('{} {}'.format(order_by, order)))
 
-    query_time = 0
+
+    ## Create a query for fastpath
+
+    fpcols = [
+        func.coalesce(0).label('m_input_no'),
+        Fastpath.measurement_start_time.label("measurement_start_time"),
+        func.concat("tid-", Fastpath.tid).label('measurement_id'),
```

Comment:
You should use the variable you have defined as `FASTPATH_MSM_ID_PREFIX` for this.